app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -90,6 +93,7 @@ public class MainActivity : AppCompatActivity(), MainViewController, Router.Call
         initReverseButton()
         centerMapOnCurrentLocation()
         presenter?.onRestoreViewState()
+        getApiKeys({initCrashReportService()})
```

Comment:
Does this mean initializing crash reports doesn't happen until the API manifest is downloaded? What if the app crashes before the download completes? Or the download itself crashes the app? I think we would still want to be able to catch those errors.

Perhaps we want to continue to inject the MINT_API_KEY via BuildConfig and reserve the API manifest for our services (Vector Tiles, Pelias, Valhalla).